### PR TITLE
Improve UX, styling, and command line functionality. Write useful readme

### PR DESCRIPTION
**Pull request by Sharpness-B**
Thanks a lot for the invitation to this org!

As I like this repo, and because it was kinda hidden and anonymous, I decided to do some functionality updates and write a useful readme.

- The package.json now provides a quick start command for getting up and running fast from the command line.
- Fixed a button that was not working
- Fixed styling of the images on the front page
- Improved UX when filtering quizzes by search
- Wrote readme

### CODE REVIEW

**Review by Ebazhanov — APPROVED** [review]
Small note maybe you can remove some commented not used code, the rest looks good to me 👍 
also I assigned @nhemnt to review your work

**Comment by nhemnt** [comment]
LGTM!

**Review by nhemnt — APPROVED** [review]

**Review by Ebazhanov — APPROVED** [review]

File: src/components/Quiz/QuizList.js [hunk 1]
```diff
@@ -55,13 +55,29 @@ const QuizList = () => {
...
+                    // constant height of images without altering with/height ratio. Center images horizontally
+                    height: 100,
+                    textAlign: "center"
+                  }}>
+                    <img
+                      className="topic-logo"
+                      src={img}
+                      alt={x.quiz_name}
+                      // width="120"
+                      // height="100"
```
Comment (Ebazhanov) [inline src/components/Quiz/QuizList.js]:
```suggestion
```

File: src/components/Quiz/QuizList.js [hunk 2]
```diff
@@ -55,13 +55,29 @@ const QuizList = () => {
...
+                      style={{
+                        // fit in frame
```
Comment (Ebazhanov) [inline src/components/Quiz/QuizList.js]:
```suggestion
```

File: src/components/Quiz/QuizList.js [hunk 3]
```diff
@@ -55,13 +55,29 @@ const QuizList = () => {
...
+                        maxHeight: 100, 
+                        maxWidth: 120,
+
+                        // center vertically
```
Comment (Ebazhanov) [inline src/components/Quiz/QuizList.js]:
```suggestion
```

File: src/components/Quiz/QuizList.js [hunk 4]
```diff
@@ -55,13 +55,29 @@ const QuizList = () => {
...
                 <Paper className={classes.paper} onClick={() => onQuizClick(x, img)}>
-                  <img
-                    className="topic-logo"
-                    src={img}
-                    alt={x.quiz_name}
-                    width="120"
-                    height="100"
-                  />
+                  <div style={{
+                    // constant height of images without altering with/height ratio. Center images horizontally
```
Comment (Ebazhanov) [inline src/components/Quiz/QuizList.js]:
```suggestion
```

**Pull request merged**